core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt

```diff
@@ -1271,9 +1272,29 @@ abstract class CoreReaderFragment :
   @OnClick(R2.id.tab_switcher_close_all_tabs)
   fun closeAllTabs() {
     closeAllTabsButton?.rotate()
+    val tempList: MutableList<KiwixWebView> = webViewList.toMutableList()
     webViewList.clear()
     tabsAdapter?.notifyDataSetChanged()
     openHomeScreen()
+    snackBarRoot?.let { root ->
+      root.bringToFront()
+      val snackbar = Snackbar.make(root, R.string.tabs_closed, Snackbar.LENGTH_LONG)
```

Comment:
Please use ```apply```  on snackbar instead of making object of snackbar.

Reply:
I am creating the object because in line 1292 I am dismissing the current snackbar and showing a new one.

Reply:
Done, should I merge both commits?